tests/models/test_openai_responses.py

```diff
@@ -255,13 +255,13 @@ async def get_image() -> BinaryContent:
                 parts=[
                     ToolReturnPart(
                         tool_name='get_image',
-                        content='See file 1.',
-                        tool_call_id='call_2hQWt32j0LvlVhi3jYuXsElZ',
+                        content='See file 1c8566e46d517d1f2c099a33f077693a9a4d986e',
+                        tool_call_id='call_4zshxmAAIg6oF12cJ56jseiQ',
                         timestamp=IsDatetime(),
                     ),
                     UserPromptPart(
                         content=[
-                            'This is file 1:',
+                            'This is file 1c8566e46d517d1f2c099a33f077693a9a4d986e:',
```

Comment:
I feel we could probably get away with much shorter IDs here, presumably would use fewer tokens.